docs/en/tutorial/tutorial_sql_2.md

```diff
@@ -36,13 +40,13 @@ In this article, we will use the main table and sub table to illustrate. We stil
 | purchase_amt  | DOUBLE    | Purchase Amount                  |
 | purchase_type | STRING    | Purchase Type: Cash, Credit Card |
 
-In the traditional relational database, in order to obtain the information of multiple tables, the most common way is to use join for splicing. However, for the requirements of Feature Engineering, the database join can not meet the requirements very efficiently. The main reason is that our main table sample table has a label column for model training, and each value can only correspond to one row of data records. Therefore, in practice, we hope that after join, the number of rows in the result table should be consistent with that in the main table.
+In the traditional relational database, the most common way to obtain the information from multiple tables is to use JOIN. However, in feature engineering, JOIN of common database can not meet the efficiency requirement. The main reason is that our main sample table has a **label** column for training, and each value of this column only correspond to one row of the data records. We hope that the number of rows in the result table should be consistent with that in the main table after JOIN operation.
 
-## 2. Sub Table Single Line Feature
+## 2. Single Line Feature in Sub Table 
```

Comment:
change to "## 2. LAST JOIN"
remove the below "## 2.1 LAST JOIN"

(as only one section, no need a sub-title of 2.1)